app/controllers/RegularContributions.scala

```diff
@@ -125,4 +126,28 @@ class RegularContributions(
         }
     case _ => Future.successful(None)
   }
+
+  def regularContributionsThankYou(): Action[AnyContent] =
```

Comment:
Since the function is inside `RegularContributions` controller, it is possible to drop the `regularContributions` from the beginning of the method.